themes/classic/assets/js/theme.js

```diff
@@ -11794,12 +11780,12 @@
 /***/ function(module, exports) {
 
 	/*!
-	 * Bootstrap v4.0.0-alpha.4 (http://getbootstrap.com)
+	 * Bootstrap v4.0.0-alpha.3 (http://getbootstrap.com)
```

Comment:
You need to run `npm update` before compiling your assets. You're downgrading Bootstrap 😢